Vagrantfile

```diff
@@ -26,23 +32,21 @@ Vagrant.configure("2") do |config|
     end
   end
 
-  config.vm.define "freebsd" do |c|
-    c.ssh.shell = "csh"
-    c.vm.box = "freebsd/FreeBSD-10.2-RELEASE"
-
+  config.vm.define "freebsd11" do |c|
+    c.vm.box = "freebsd/FreeBSD-11.1-RELEASE"
+    c.vm.guest = :freebsd
+    c.vm.hostname = "freebsd11"
+    c.vm.base_mac = "6658695E16F0"
     c.vm.network "private_network", type: "dhcp"
-    c.vm.synced_folder ".", "/vagrant", type: "nfs"
+    c.vm.synced_folder ".", "/vagrant", disabled: true
```

Comment:
The vagrant box isn't used just for building, it can be used for debugging too. If I'm debugging I absolutely want a shared folder so that I can edit the files in the vm using the editor on the host.